packages/ckeditor5-image/tests/image/replaceimagesourcecommand.js

```diff
@@ -44,6 +44,33 @@ describe( 'ReplaceImageSourceCommand', () => {
 
 			expect( element.getAttribute( 'src' ) ).to.equal( 'bar/foo.jpg' );
 		} );
+
+		it( 'should clean up some attributes in responsive image', () => {
+			setModelData( model, `[<imageBlock
+				src="foo/bar.jpg"
+				ckboxImageId="id"
```

Comment:
This test data does not really belong to `ckeditor5-image`. I would test the specific case in `ckbox` and add here (in `image`) tests of the cleanup mechanism.